api/proto/common/v2/common_v2.proto

```diff
@@ -36,12 +34,15 @@ message BlobHeader {
 message BlobCertificate {
   // blob_header contains data about the blob.
   BlobHeader blob_header = 1;
-  // relays is the list of relays that are in custody of the blob.
+  // signature is an ECDSA signature signed by the blob request signer's account ID over the BlobHeader's blobKey,
+  // which is a keccak hash of the serialized BlobHeader, and used to verify against blob dispersal request's account ID
```

Comment:
How do you feel about removing blobKey everywhere and using the more explicit blobHeaderHash?
Explicit self-documenting names prevent having to add this kind of detail in every comment everywhere.

Reply:
Don't think it does, because it literally plays no purpose. It's a symlink that shouldn't exist. An indirection level that is not helpful. But I'm losing the battle here so will resolve.